orderbook/Cargo.toml

```diff
@@ -4,10 +4,12 @@ version = "0.1.0"
 edition = "2018"
 
 [dependencies]
-anyhow = "1.0.33"
-ethcontract = { version = "0.8.0", default-features = false }
-hex-literal = "0.3.1"
+anyhow = "1.0"
```

Comment:
Here I've unified the version number to always be major and minor without patch.